Servers/UI/OJS.Servers.Ui/ClientApp/src/pages/contests/contest-register/ContestRegister.tsx

```diff
@@ -66,6 +66,9 @@ const ContestRegister = () => {
     }, [ isLoading, isRegisteredSuccessfully, navigate, contestId, participationType, shouldConfirmParticipation, requirePassword ]);
 
     const renderContestRegisterBody = useCallback(() => {
+        if (!isRegisteredSuccessfully && !requirePassword && !shouldConfirmParticipation) {
+            return (<div>You are not registered for this exam!</div>);
+        }
```

Comment:
Will this message appear if the user is not added in the Exam Group for the contest?

Reply:
This is a change made by @BojidarTonev. He handles this better in his other changes so i guess this will be deleted later.